features/dual-stack-support.md

```diff
@@ -0,0 +1,229 @@
+**Relevant Documentation:**
+
+[Dual Stack support](https://github.com/istio/istio/issues/40394)
+
+[//]: # (Links to relevant documentation for feature)
+
+**RFC:**
+
+[Original design solution for Dual Stack support in Istio](https://docs.google.com/document/d/1oT6pmRhOw7AtsldU0-HbfA0zA26j9LYiBD_eepeErsQ/edit?usp=sharing)
+
+[//]: # (Link to RFC for feature)
+
+
+---
+
+## Experimental
+
+### Requirements:
+
+[//]: # (All information in this section is mandatory for promotion. Please modify the links in this)
+[//]: # (section.)
+
+- [x] [User stories](https://docs.google.com/document/d/15LP2XHpQ71ODkjCVItGacPgzcn19fsVhyE7ruMGXDyU/edit?usp=sharing) reviewed in a work group meeting.
+
+[//]: # (User stories are a way to communicate user value. User stories follow the style)
+[//]: # (as a [type of user], I want [an action] so that [a benefit/a value]. Istio currently has no user)
+[//]: # (story template. Maybe you can make one?)
+
+[//]: # (User stories must be presented in a work group meeting. They need no approval and are later integrated)
+[//]: # (into the RFCs, which do need approval for alpha. You may find value to negotiate within the work group where the)
+[//]: # (user stories are presented to help clarify the user stories.)
+
+- [x] [RFC Authored] - [Supporting Dual Stack](https://docs.google.com/document/d/15LP2XHpQ71ODkjCVItGacPgzcn19fsVhyE7ruMGXDyU/edit?usp=sharing).
+
+[//]: # (An RFC is mandatory to graduate to experimental. The RFC does not have to be reviewed in a work group)
+[//]: # (meeting to graduate to experimental.)
+
+- [x] [Documentation](https://istio.io/latest/news/releases/1.17.x/announcing-1.17/#istio-dual-stack-support), as well as [Supporting Dual Stack](https://docs.google.com/document/d/15LP2XHpQ71ODkjCVItGacPgzcn19fsVhyE7ruMGXDyU/edit?usp=sharing) for enabling and using the feature.
+
+[//]: # (The documentation instructions may exist on the developer wiki or the team drive. They may include instructions)
+[//]: # (for building running a `istioctl experimental command`, or using the preview profile,)
+[//]: # (or any other relevant information.)
+
+- [x] [Feedback plan](insert_your_link_here). N/A, we already have feedback from users that this is desired and useful.
+
+[//]: # (This may include user feedback meetings, discuss.istio.io conversations, GitHub issues, or mailing lists.)
+
+- [x] Disabled by default.
+
+- [x] No impact on performance when the feature is disabled.
+
+
+[//]: # (Once all other items are completed, features.yaml should be updated to promote the feature)
+
+- [x] [features.yaml](https://github.com/istio/enhancements/blob/master/features.yaml) updated for this feature
+---
+
+## Alpha
+
+### Requirements: 
+
+**Design**
+
+- [ ] RFC has been approved describing the intention of the feature as well as the user stories behind the feature. 
+N/A
+
+**Config**
+
+- [x] Explicit user action is required to enable this feature (e.g. a config field, config resource, or installation action). 
```

Comment:
Do you have an istio.io document on how to enable the feature and use? If so, please link it here?

Reply:
Hi @kfaseela, according to @jacob-delgado, there should be other blog for experiment branch and master branch installation. However, there is the installation step in the optimized design doc. I will add the proposal link here

Reply:
done

Reply:
@zhlsunshine an example of an experimental feature on istio.io that may help: https://istio.io/latest/docs/tasks/security/cert-management/custom-ca-k8s/